optimum/intel/openvino/quantization.py

```diff
@@ -312,10 +312,12 @@ def start_async(
                 self,
                 inputs: Any = None,
                 userdata: Any = None,
-                shared_memory: bool = False,
+                share_inputs: bool = False,
+                *,
+                shared_memory: Any = None,
             ):
                 data_cache.append(inputs)
-                self.request.infer(inputs, shared_memory)
+                self.request.infer(inputs, share_inputs, share_outputs=True, shared_memory=shared_memory)
```

Comment:
@AlexKoff88, this is how I have fixed it. Please provide feedback if something looks weird from your perspective.